config/environments/production.rb

```diff
@@ -52,6 +52,9 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
   config.ssl_options = {hsts: {subdomains: false}}
+  if ENV["FORCE_SSL"] == "false"
+    config.force_ssl = false
+  end
```

Comment:
I think it might be better to move this up to line 53 so the config always happens in the same place.

How about:

```ruby
config.force_ssl = (ENV["FORCE_SSL"] == "false") ? false : true
```

Reply:
That seems cleaner indeed!